.markdownlint.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
good idea

but maybe it's necessary to add package.json?

Reply:
removed, need time to research and add good some lint tool